### PR TITLE
add ampINJ, ampJUNO, ampSEI

### PR DESCRIPTION
**Pull request by 0xphilipp**

### CODE REVIEW

File: sei/assetlist.json [hunk 1]
```diff
@@ -44,11 +43,28 @@
...
+                    "denom": "ampSEI",
+                    "exponent": 6
+                }
+            ],
+            "base": "factory/sei1x2fgaaqecvk8kwuqkjqcj27clw5p5g99uawdzy9sc4rku8avumcq3cky4k/ampSEI",
+            "name": "ERIS Amplified SEI",
+            "display": "ampSEI",
+            "symbol": "ampSEI",
+            "logo_URIs": {
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sei/images/ampSEI.svg"
```
Comment (JeremyParish69) [inline sei/assetlist.json:66]: for ampJUNO and ampSEI, the images are titled ampJuno and ampSei, but the image references have ampJUNO and ampSEI. Please note the discrepancy by the capital letters and realign the image names.

**Comment by 0xphilipp** [comment]
Ok @JeremyParish69 its ready for review. I had some issues with casing changes with git, but all fixed now.

**Comment by JeremyParish69** [comment]
Sorry, @0xphilipp , it looks like there are conflicts now and I don't have permissions to resolve them myself

**Comment by 0xphilipp** [comment]
Fixed conflicts @JeremyParish69

**Pull request merged**